src/box/errcode.h

```diff
@@ -141,287 +141,313 @@ struct errcode_record {
+	_(ER_SQL_NO_SUCH_PRAGMA, 175,		"Pragma '%s' does not exist", "pragma", STRING) \
+	_(ER_SQL_CANT_RESOLVE_FIELD, 176,	"Can't resolve field '%s'", "field", STRING) \
+	_(ER_INDEX_EXISTS_IN_SPACE, 177,	"Index '%s' already exists in space '%s'", "index", STRING, "space", STRING) \
+	_(ER_INCONSISTENT_TYPES, 178,		"Inconsistent types: expected %s got %s", "expected", STRING, "actual", STRING) \
+	_(ER_SQL_SYNTAX_WITH_POS, 179,		"Syntax error at line %d at or near position %d: %s", "line", INT, "column", INT, "details", STRING) \
 	_(ER_SQL_STACK_OVERFLOW, 180,		"Failed to parse SQL statement: parser stack limit reached") \
 	_(ER_SQL_SELECT_WILDCARD, 181,		"Failed to expand '*' in SELECT statement without FROM clause") \
 	_(ER_SQL_STATEMENT_EMPTY, 182,		"Failed to execute an empty SQL statement") \
-	_(ER_SQL_KEYWORD_IS_RESERVED, 183,	"At line %d at or near position %d: keyword '%.*s' is reserved. Please use double quotes if '%.*s' is an identifier.") \
-	_(ER_SQL_SYNTAX_NEAR_TOKEN, 184,	"Syntax error at line %d near '%.*s'") \
-	_(ER_SQL_UNKNOWN_TOKEN, 185,		"At line %d at or near position %d: unrecognized token '%.*s'") \
-	_(ER_SQL_PARSER_GENERIC, 186,		"%s") \
-	_(ER_SQL_ANALYZE_ARGUMENT, 187,		"ANALYZE statement argument %s is not a base table") \
-	_(ER_SQL_COLUMN_COUNT_MAX, 188,		"Failed to create space '%s': space column count %d exceeds the limit (%d)") \
-	_(ER_HEX_LITERAL_MAX, 189,		"Hex literal %s%s length %d exceeds the supported limit (%d)") \
-	_(ER_INT_LITERAL_MAX, 190,		"Integer literal %s%s exceeds the supported range [-9223372036854775808, 18446744073709551615]") \
-	_(ER_SQL_PARSER_LIMIT, 191,		"%s %d exceeds the limit (%d)") \
-	_(ER_INDEX_DEF_UNSUPPORTED, 192,	"%s are prohibited in an index definition") \
-	_(ER_CK_DEF_UNSUPPORTED, 193,		"%s are prohibited in a ck constraint definition") \
-	_(ER_MULTIKEY_INDEX_MISMATCH, 194,	"Field %s is used as multikey in one index and as single key in another") \
-	_(ER_CREATE_CK_CONSTRAINT, 195,		"Failed to create check constraint '%s': %s") \
-	_(ER_CK_CONSTRAINT_FAILED, 196,		"Check constraint failed '%s': %s") \
-	_(ER_SQL_COLUMN_COUNT, 197,		"Unequal number of entries in row expression: left side has %u, but right side - %u") \
-	_(ER_FUNC_INDEX_FUNC, 198,		"Failed to build a key for functional index '%s' of space '%s': %s") \
-	_(ER_FUNC_INDEX_FORMAT, 199,		"Key format doesn't match one defined in functional index '%s' of space '%s': %s") \
-	_(ER_FUNC_INDEX_PARTS, 200,		"Wrong functional index definition: %s") \
-	_(ER_NO_SUCH_FIELD_NAME, 201,		"Field '%s' was not found in the tuple") \
-	_(ER_FUNC_WRONG_ARG_COUNT, 202,		"Wrong number of arguments is passed to %s(): expected %s, got %d") \
+	_(ER_SQL_KEYWORD_IS_RESERVED, 183,	"At line %d at or near position %d: keyword '%s' is reserved. Please use double quotes if '%s' is an identifier.", \
+						"line", INT, "column", INT, "keyword", STRING, "", STRING) \
```

Comment:
I think we should use "position" instead of "column" here and in other SQL errors.

Reply:
Fixed.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index e52f87c30..bb9bd22b0 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -330,14 +330,14 @@ struct errcode_record {
        _(ER_SQL_CANT_RESOLVE_FIELD, 176,       "Can't resolve field '%s'", "field", STRING) \
        _(ER_INDEX_EXISTS_IN_SPACE, 177,        "Index '%s' already exists in space '%s'", "index", STRING, "space", STRING) \
        _(ER_INCONSISTENT_TYPES, 178,           "Inconsistent types: expected %s got %s", "expected", STRING, "actual", STRING) \
-       _(ER_SQL_SYNTAX_WITH_POS, 179,          "Syntax error at line %d at or near position %d: %s", "line", INT, "column", INT, "details", STRING) \
+       _(ER_SQL_SYNTAX_WITH_POS, 179,          "Syntax error at line %d at or near position %d: %s", "line", INT, "position", INT, "details", STRING) \
        _(ER_SQL_STACK_OVERFLOW, 180,           "Failed to parse SQL statement: parser stack limit reached") \
        _(ER_SQL_SELECT_WILDCARD, 181,          "Failed to expand '*' in SELECT statement without FROM clause") \
        _(ER_SQL_STATEMENT_EMPTY, 182,          "Failed to execute an empty SQL statement") \
        _(ER_SQL_KEYWORD_IS_RESERVED, 183,      "At line %d at or near position %d: keyword '%s' is reserved. Please use double quotes if '%s' is an identifier.", \
-                                               "line", INT, "column", INT, "keyword", STRING, "", STRING) \
+                                               "line", INT, "position", INT, "keyword", STRING, "", STRING) \
        _(ER_SQL_SYNTAX_NEAR_TOKEN, 184,        "Syntax error at line %d near '%s'", "line", INT, "token", STRING) \
-       _(ER_SQL_UNKNOWN_TOKEN, 185,            "At line %d at or near position %d: unrecognized token '%s'", "line", INT, "column", INT, "value", STRING) \
+       _(ER_SQL_UNKNOWN_TOKEN, 185,            "At line %d at or near position %d: unrecognized token '%s'", "line", INT, "position", INT, "value", STRING) \
        _(ER_SQL_PARSER_GENERIC, 186,           "%s", "details", STRING) \
        /* ER_SQL_ANALYZE_ARGUMENT, 187, Unused */ \
        _(ER_SQL_COLUMN_COUNT_MAX, 188,         "Failed to create space '%s': space column count %d exceeds the limit (%d)", "space", STRING, "value", INT, "max", INT) \
@@ -360,7 +360,7 @@ struct errcode_record {
        _(ER_BOOTSTRAP_READONLY, 203,           "Trying to bootstrap a local read-only instance as master") \
        _(ER_SQL_FUNC_WRONG_RET_COUNT, 204,     "SQL expects exactly one argument returned from %s, got %d", "func", STRING, "actual", INT)\
        _(ER_FUNC_INVALID_RETURN_TYPE, 205,     "Function '%s' returned value of invalid type: expected %s got %s", "func", STRING, "expected", STRING, "actual", STRING) \
-       _(ER_SQL_PARSER_GENERIC_WITH_POS, 206,  "At line %d at or near position %d: %s", "line", INT, "column", INT, "details", STRING) \
+       _(ER_SQL_PARSER_GENERIC_WITH_POS, 206,  "At line %d at or near position %d: %s", "line", INT, "position", INT, "details", STRING) \
        _(ER_REPLICA_NOT_ANON, 207,             "Replica '%s' is not anonymous and cannot register.", "replica", STRING) \
        _(ER_CANNOT_REGISTER, 208,              "Couldn't find an instance to register this replica on.") \
        _(ER_SESSION_SETTING_INVALID_VALUE, 209, "Session setting %s expected a value of type %s", "setting", STRING, "expected", STRING) \
```

</details>